packages/apps/auth0-actions/src/helpers.ts

```diff
@@ -0,0 +1,26 @@
+import { PatronRecord, SierraClient } from '@weco/sierra-client';
+import { User } from 'auth0';
+import { ResponseStatus } from '@weco/identity-common';
+
+export const patronRecordToUser = (patronRecord: PatronRecord): User => ({
+  user_id: 'p' + patronRecord.recordNumber,
+  email: patronRecord.email,
+  name: patronRecord.firstName + ' ' + patronRecord.lastName,
+  given_name: patronRecord.firstName,
+  family_name: patronRecord.lastName,
+});
+
+export const recordNumberForEmail = async (
+  email: string,
+  sierraClient: SierraClient
+): Promise<number | undefined> => {
+  const patronRecordResponse = await sierraClient.getPatronRecordByEmail(email);
+
+  if (patronRecordResponse.status === ResponseStatus.Success) {
+    return patronRecordResponse.result.recordNumber;
+  } else if (patronRecordResponse.status === ResponseStatus.NotFound) {
+    return undefined;
```

Comment:
I wonder if this should be an error also?

```suggestion
throw new Error(`Could not find user with email address [${email}]`)
```

So errors are returned consistently.

Reply:
You'd think so, but as per [the documentation of the callback values](https://auth0.com/docs/connections/database/custom-db/templates/change-email), we should be returning...
> a null first parameter with no corresponding second parameter (or one with a value of false) indicates that no password change was performed (possibly due to the user not being found)

Reply:
Could you put a comment to that effect/link to that documentation?